core/src/net/sf/openrocket/util/TestRockets.java

```diff
@@ -524,6 +524,24 @@ public static final Rocket makeEstesAlphaIII(){
 		return rocket;
 	}
 	
+	// This function is used for unit, integration tests, DO NOT CHANGE (without updating tests).
+	public static final void splitRocketFins( BodyTube body, FinSet fins, int finCount){
+		// actually remove the fins
+		body.removeChild(fins);
+
+		FinSet template = fins; 
+		template.setFinCount(1);
+		// and manually add in the equivalent:
+		for(int finNumber=0; finNumber<finCount; ++finNumber){
+			final TrapezoidFinSet singleFin = new TrapezoidFinSet(1, 0.05, 0.03, 0.02, 0.05);
```

Comment:
Could the parameters on these fins come from copying the parameters from the original FinSet?

Reply:
It gave me more control, But you make a good point.  I'll convert it.

Reply:
tyg.